NEWS.md

```diff
@@ -1,5 +1,10 @@
 # NEWS
 
+## 0.8.18 (in development)
```

Comment:
Your NEWS is pretty out-of-date. Current in-dev version is 0.8.25. The current default branch is main, right?